tests/k8s/1-001_validate_basic/01-assert.yaml

```diff
@@ -10,6 +10,11 @@ metadata:
   name: example-argocd
 status:
   phase: Available
+  conditions:
```

Comment:
I tried it before, but in existing logic it is not possible to intentionally fail the reconciliation. At few places we can make invalid configs in CR, which will log error, but operator doesn't return it, since in next cycle it will fail again until user changes the CR configs. I checked existing test cases as well, but there are no negative test cases.